app/models/user.rb

```diff
@@ -43,10 +51,30 @@ def group
     end
   end
 
+  def hastus_id
+    badge_number
+  end
+
+  def inactive_message
+    if !active? then :inactive
+    else super
+    end
+  end
+
   def proper_name
     [last_name, first_name].join ', '
   end
 
+  def requires_password_change?
+    !(driver? || password_changed_from_default?)
```

Comment:
That is, drivers and people who have changed their password, do not need to change their password.

Reply:
How do you tell? Is this a devise thing?

Reply:
https://github.com/umts/incidents/pull/89#discussion_r158549973